scripts/run_e2e_tests.py

```diff
@@ -265,22 +311,61 @@ def run_tests(args):
                 'PORTSERVER_ADDRESS': common.PORTSERVER_SOCKET_FILEPATH,
             }))
 
-        stack.enter_context(servers.managed_webdriver_server(
-            chrome_version=args.chrome_driver_version))
+        if args.suite == 'full':
+            stack.enter_context(servers.managed_webdriver_server(
+                chrome_version=args.chrome_driver_version))
+
+            proc = stack.enter_context(servers.managed_protractor_server(
+                suite_name=args.suite,
+                dev_mode=dev_mode,
+                debug_mode=args.debug_mode,
+                sharding_instances=args.sharding_instances,
+                stdout=subprocess.PIPE))
+
+            proc = stack.enter_context(servers.managed_webdriverio_server(
+                suite_name=args.suite,
+                dev_mode=dev_mode,
+                debug_mode=args.debug_mode,
+                chrome_version=args.chrome_driver_version,
+                sharding_instances=args.sharding_instances,
+                stdout=subprocess.PIPE))
+
+        elif args.suite in SUITES_MIGRATED_TO_WEBDRIVERIO:
+            proc = stack.enter_context(servers.managed_webdriverio_server(
```

Comment:
We are doing it in the [managed_webdriverio_server](https://github.com/oppia/oppia/blob/f94aa7b76f4d6e52f70f9129b0d1a4451d678e93/scripts/servers.py#L708). I am not using the manage_webdriverio_server function because it starts the selenium server as well after installing the chrome driver which we dont want as it will not allow us to run the test in debug mode for webdriverio.

Reply:
We are doing it [here](https://github.com/oppia/oppia/pull/15614/files#diff-377b44d33bdb8f191bd855ce9ef4adf8ce1f42bde7ce5edc3d0a10a32ac7dfe9R708)